docs/bos/queryapi/context.md

```diff
@@ -0,0 +1,348 @@
+---
+id: context-object
+title: QueryAPI Context object
+sidebar_label: Context object
+---
+
+## Overview
+
+The `context` object is a global object made available to developers building indexers with [QueryAPI](intro.md). It provides helper methods for developers to interact with things outside their coding environment. Crucially, these involve the database and GraphQL endpoint spun up along with the indexer, as well as allowing API calls. 
+
+:::caution Under development
+
+Many of the formatting and changes in this document are still in progress. Once these changes are merged, they will be available in Dev. Specifically, [auto-complete](#auto-complete) and the `context.db.TableName.methodName` format. 
+
+:::
+
+## Main Methods
+
+:::tip
+Most methods are asynchronous, hence why all examples have the `await` keyword in front of the function call. 
```

Comment:
Actually all of them are. I double checked.

Reply:
fixed